uk.co.compucorp.civicrm.hrcore/CRM/HRCore/Upgrader.php

```diff
@@ -188,6 +189,30 @@ private function createDefaultRelationshipTypes() {
     }
   }
 
+  /**
+   * Making All Currencies Available for new installations
+   */
+  private function makeAllCurrenciesAvailable() {
+    $dao = CRM_Core_DAO::executeQuery('SELECT * from civicrm_currency');
+    while ($dao->fetch()) {
+      $currencyExists = civicrm_api3('OptionValue', 'get', [
+        'value' => $dao->name,
+      ]);
+      if (empty($currencyExists['values'])) {
+        civicrm_api3('OptionValue', 'create', [
+          'option_group_id' => 'currencies_enabled',
+          'label' => $dao->name . ' (' . $dao->symbol . ')',
+          'value' => $dao->name,
+          'name' => $dao->name,
+          'is_active' => 1,
```

Comment:
I don't think we need to se is_active, is_reserved, is_optgroup and is_default, The values set for them are the same they have as the default database values.